.eslintrc.js

```diff
@@ -102,22 +102,26 @@ module.exports = {
     },
     {
       // edit rules here to modify test linting
-      files: ['__tests__/**', '*.test.ts', '**/amplify-e2e-tests/**'],
-      plugins: ['jest'],
-      extends: ['plugin:jest/recommended'],
-      rules: {
-        '@typescript-eslint/unbound-method': 'off',
-        'jest/unbound-method': 'error',
-        '@typescript-eslint/no-explicit-any': 'off',
-        'spellcheck/spell-checker': 'off',
+      files: ['**/__tests__/**', '**/__test__/**', '*.test.ts', '**/amplify-e2e-*/**', '**/test/**', '**/tests/**', '**/__e2e__/**'],
+      parserOptions: {
+        project: ['tsconfig.tests.json'],
```

Comment:
Override of `tsconfig` is required. The `tsconfig` used by parser must include sources that linter attempts to process. The `tsconfig` used for non-test codebase excludes tests.
Otherwise content of this `tsconfig` does not matter much for the rules we care about.